src/app/functions/server/getBreaches.ts

```diff
@@ -5,40 +5,28 @@
 import { logger } from "./logging";
 import {
   HibpLikeDbBreach,
-  formatDataClassesArray,
   getAllBreachesFromDb,
-  req,
-} from "../../../utils/hibp.js";
+  fetchHibpBreaches,
+} from "../../../utils/hibp";
 import { upsertBreaches } from "../../../db/tables/breaches.js";
-import { Breach } from "../universal/breach.js";
 
-let breaches: Array<Breach | HibpLikeDbBreach>;
+let breaches: Array<HibpLikeDbBreach>;
 
-export async function getBreaches() {
+export async function getBreaches(): Promise<HibpLikeDbBreach[]> {
   if (breaches) {
     return breaches;
   }
   breaches = await getAllBreachesFromDb();
-  logger.debug(
-    "loadBreachesIntoApp",
-    `loaded breaches from database: ${breaches.length}`,
-  );
+  logger.debug(`loaded breaches from database: ${breaches.length}`);
```

Comment:
It'll be easier to find in the logs if you use it like:

```
logger.debug("loaded breaches from database", { breachesLength: breaches.length });
```

`loaded breaches from database` will be a key that is indexed and fast to search in the log console.

Reply:
The convention is that we now use snake case for the first param, so probably something like
`loaded_breaches_from_database`

Reply:
Heh, I just removed the mention of the unused function, but fixed in 9e4191322.